src/modules/commander/commander.cpp

```diff
@@ -2881,7 +2881,7 @@ int commander_thread_main(int argc, char *argv[])
 						       _mission_result.stay_in_failsafe,
 						       &status_flags,
 						       land_detector.landed,
-						       (rc_loss_enabled > 0),
+						       rc_loss_mode,
```

Comment:
I read more commander and navigator code and now feel the difference of their responsibilities. 
(Please correct me if I am wrong)
Navigator simply performs 'pieces' of the autonomous navigation, it doesn't make a decision.
Commander is that one that maintains the global state of the vehicle based on params, user and sensor inputs and makes all the decisions of what action to take in particular situation.

If my understanding is correct then I am 100% agree that rc_loss_mode should be handled in commander (SM helper), not navigator.

My only concern is that SM helper keeps growing and already has a big chunk of copy-pasted code. I feel a bit worried of breaking things by doing change to RC loss like this without having really good understanding and having unit tests covering the logic.